fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexTest.java

```diff
@@ -1052,22 +1054,11 @@ void searchForAutoCompleteCrossingMultipleFields(boolean withTermVectors) throws
 
             // Assert the suggestions' keys
             List<String> suggestions = results.stream().map(i -> (String) i.getKey().get(i.getKeySize() - 1)).collect(Collectors.toList());
-            assertEquals(ImmutableList.of("good evening", "Good night", "Good morning", "Good afternoon", "I'm good", "That's really good!"), suggestions);
+            assertThat(suggestions, containsInAnyOrder("good evening", "Good night", "Good morning", "Good afternoon", "I'm good", "That's really good!"));
 
             // Assert the corresponding field for the suggestions
             List<String> fields = results.stream().map(i -> (String) i.getKey().get(i.getKeySize() - 2)).collect(Collectors.toList());
             assertEquals(ImmutableList.of("text", "text", "text", "text", "text2", "text2"), fields);
-
-            // Assert the suggestions are sorted according to their values, which are determined by the position of the term into the indexed text
```

Comment:
The results are now returned in the same order as the original Lucene query, so these asserts sorting by score is no longer true. If we wanted to continue this, we would need to add some kind of sort step, which might actually be possible given that we have bounded results. But I haven't done that in this PR